phalcon/Mvc/Collection/ManagerInterface.zep

```diff
@@ -41,56 +41,56 @@ interface ManagerInterface
     /**
      * Binds a behavior to a collection
      */
-    public function addBehavior(<CollectionInterface> model, <BehaviorInterface> behavior);
+    public function addBehavior(<CollectionInterface> collection, <BehaviorInterface> behavior);
 
     /**
-     * Returns the connection related to a model
+     * Returns the connection related to a collection
      */
-    public function getConnection(<CollectionInterface> model) -> <AdapterInterface>;
+    public function getConnection(<CollectionInterface> collection) -> <AdapterInterface>;
 
     /**
-     * Returns a custom events manager related to a model
+     * Returns a custom events manager related to a collection
      */
-    public function getCustomEventsManager(<CollectionInterface> model) -> <EventsManagerInterface>;
+    public function getCustomEventsManager(<CollectionInterface> collection) -> var | null;
```

Comment:
I propose change this to 
```suggestion
    public function getCustomEventsManager(<CollectionInterface> collection) -> <EventsManagerInterface> | null;
```

as well as https://github.com/phalcon/cphalcon/blob/3addad41e14ee83d6bb76138fb4706cadff50182/phalcon/Mvc/Collection/Manager.zep#L92